CSharp/TextSearch/Common/Helpers.cs

```diff
@@ -38,7 +38,7 @@ public static string GetSplitFileName(int fileNumber)
         /// <param name="boundTask">The task.</param>
         /// <param name="dumpStandardOutOnTaskSuccess">True to log the standard output file of the task even if it succeeded.  False to not log anything if the task succeeded.</param>
         /// <returns>The string containing the standard out of the file, or null if stdout could not be gathered.</returns>
-        public static async Task<string> CheckForTaskSuccessAsync(CloudTask boundTask, bool dumpStandardOutOnTaskSuccess)
+        public static async Task CheckForTaskSuccessAsync(CloudTask boundTask, bool dumpStandardOutOnTaskSuccess)
```

Comment:
Shouldn't this return something?

Reply:
It doesn't have to, it can just raise if the task failed.

I need to fix the comment though.